src/fastpm.c

```diff
@@ -168,13 +174,15 @@ int main(int argc, char ** argv) {
 
     fastpm_info("np_alloc_factor = %g\n", CONF(prr->lua, np_alloc_factor));
 
+    FastPMCosmology * cosmology = malloc(sizeof(cosmology[0]));
+    prepare_cosmology(cosmology, prr);
```

Comment:
@rainwoodman is preparing the cosmology object in this way ok? Or is cosmology[1] better?